libs/scraper-archive/src/snapshots.ts

```
@@ -0,0 +1,90 @@
+      frame: undefined,
+    }
+  }
+  return r;
+}
+
+export type SnapshotMeta = {
+  timestamp: Date;
+  filename: string;
+}
+
+export function getSnapshots(folder: string, element: string, limit=100): SnapshotMeta[] {
```

ESLint

[error] Operator '=' must be spaced.

(space-infix-ops)